paddle_image.py

```diff
@@ -0,0 +1,130 @@
+        ``se_resnext101_32x4d_imagenet``,  ``resnext50_vd_64x4d_imagenet``,
+        ``resnext50_vd_32x4d_imagenet``, `resnext50_64x4d_imagenet``,
+        ``resnext50_32x4d_imagenet``, ``resnext152_vd_64x4d_imagenet``,
+        ``resnext152_64x4d_imagenet``, ``resnext152_32x4d_imagenet``,
+        ``resnext101_vd_64x4d_imagenet``, ``resnext101_vd_32x4d_imagenet``,
+        ``resnext101_32x8d_wsl``, ``resnext101_32x48d_wsl``, ``resnext101_32x32d_wsl``,
+        ``resnext101_32x16d_wsl``, ``resnet_v2_50_imagenet``, ``resnet_v2_34_imagenet``,
+        ``resnet_v2_18_imagenet``, ``resnet_v2_152_imagenet``, ``resnet_v2_101_imagenet``,
+        ``mobilenet_v2_imagenet``, ``inception_v4_imagenet``, ``googlenet_imagenet``,
+        ``efficientnetb7_imagenet``, ``efficientnetb6_imagenet``, ``efficientnetb5_imagenet``,
+        ``efficientnetb4_imagenet``, ``efficientnetb3_imagenet``, ``efficientnetb2_imagenet``,
+        ``efficientnetb1_imagenet``, ``efficientnetb0_imagenet``, ``dpn68_imagenet``,
+        ``dpn131_imagenet``, ``dpn107_imagenet``, ``densenet264_imagenet``,
+        ``densenet201_imagenet``, ``densenet169_imagenet``, ``densenet161_imagenet``,
+        ``densenet121_imagenet``, ``darknet53_imagenet``, ``alexnet_imagenet``,
+    :param pool_strategy: the pooling strategy. Default is `None`.
+    :param channel_axis: The axis of the color channel, default is -3
+    :param default_batch_size: size of each batch
+    :param default_traversal_paths: traversal path of the Documents, (e.g. 'r', 'c')
+    :param on_gpu: set to True if using GPU
+    :param args:  Additional positional arguments
+    :param kwargs: Additional keyword arguments
+    """
+    def __init__(
+            self,
+            model_name: str = 'xception71_imagenet',
+            pool_strategy: str = 'mean',
+            channel_axis: int = -3,
+            default_batch_size: int = 32,
+            default_traversal_paths: Union[str, List[str]] = 'r',
+            on_gpu: bool = False,
+            *args,
+            **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        self.pool_strategy = pool_strategy
+        self.channel_axis = channel_axis
+        self.model_name = model_name
+        self._default_channel_axis = -3
+        self.inputs_name = None
+        self.outputs_name = None
+        self.on_gpu = on_gpu
+        self.default_batch_size = default_batch_size
+        self.default_traversal_paths = default_traversal_paths
+
+        import paddlehub as hub
+        module = hub.Module(name=self.model_name)
+        inputs, outputs, self.model = module.context(trainable=False)
+        self._get_inputs_and_outputs_name(inputs, outputs)
+
+        import paddle.fluid as fluid
+        self.device = fluid.CUDAPlace(0) if self.on_gpu else fluid.CPUPlace()
+        self.exe = fluid.Executor(self.device)
+
+    @requests
+    def encode(self, docs: DocumentArray, parameters: dict, **kwargs) -> DocumentArray:
```

Comment:
Function returns `None` actually

Reply:
yes, it should not be anotated with this `return`

Reply:
good  catch!